tools/tests/form-control/validation.test.js

```diff
@@ -118,3 +253,96 @@ describe('Form validation', () => {
         });
     });
 });
+
+describe('Form custom validation', () => {
+    afterAll(() => {
+        const tooltips = document.querySelectorAll('gameface-tooltip');
+        for (let i = 0; i < tooltips.length; i++) {
+            tooltips[i].parentElement.removeChild(tooltips[i]);
+        }
+
+        let currentElement = document.querySelector('.form-validation-test-wrapper');
+
+        if (currentElement) {
+            currentElement.parentElement.removeChild(currentElement);
+        }

```

Comment:
Rebase when ready and use the new helper method for that^

Reply:
Replaced with the method